### PR TITLE
Implemented SetFileTime

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

**Comment by akallabeth** [comment]
@freerdp-bot Stop kidding me and test this!

**Comment by mfleisz** [comment]
LGTM

**Comment by realjiangms** [comment]
Is the NATIVE_API_LEVEL change on android expected?

**Comment by hardening** [comment]
Looks good

**Comment by akallabeth** [comment]
@realjiangms Yes, android-9 is lacking too many posix APIs making it a nightmare to maintain.

File: scripts/android-build.conf [hunk 1]
```diff
@@ -4,10 +4,11 @@
 SCRIPT_PATH=$(dirname "${BASH_SOURCE[0]}")
 SCRIPT_PATH=$(realpath "$SCRIPT_PATH")
 
-WITH_JPEG=1
+WITH_JPEG=0
```
Comment (DavBfr) [inline scripts/android-build.conf]: This has nothing to do with this patch

**Comment by akallabeth** [comment]
@DavBfr you are right, I've incremented for missing defines. but the function is still missing. will revert tomorrow.

**Comment by akallabeth** [comment]
@DavBfr Oh, sorry mixed up the line of your comment. Disabled this as it is not used on android and the build bot is a bit picky. You're right that it is not directly related to this pull, but as it changes android specific code I added that.

**Pull request merged**